content/_layouts/base.njk

```diff
@@ -45,14 +45,30 @@
     <script>try{Typekit.load({ async: true });}catch(e){}</script>
   </head>
 
-  <body
-    data-layout="body">
-    <a href="#content" class="to-content">Skip to main content</a>
-    {% include "site/menu.njk" %}
-    {% include 'site/banner.njk' %}
+  <body data-layout="body">
+    <a href="#page-title"
+      id="to-content"
+      class="sr-only">
+      Skip to main
+    </a>
+
+    {% include "site/banner.njk" %}
+
+    <main data-layout="main">
+      <header class="title">
+        <h1 class="title-main"
+          id="page-title"
+          tabindex="-1">
+          {{ full_title | safe }}
+        </h1>
+
+        {% if sub %}
+          <p class="subtitle">
```

Comment:
All the articles I find are somewhat old, but suggest that `h1-h6` are *not* to be used for  subtitles, since a subtitle is not the same as a nested section header. Most recommend a `header` around `h1-h6` title and `p` subtitle.

Reply:
that is unfortunate. I feel like some of our subtitles are more important than the other headlines on a page...and some pages it is a continuation of the article title (ie. https://www.oddbird.net/2019/11/25/shape-editor/). I wonder if we want an option to add a span in the h1 and wrap the whole thing in those cases.

Reply:
Yeah, we can certainly do that. I'm not actually sure how it balances out. The `p` is inside a `header`, so that does give it some weight. Putting it inside the `h1` might give it more weight, but does it also make the headings less skim-able? Can we make sure the heading stands alone when skimming, even if the tagline adds a lot of context?

Reply:
I imagined the span would have  the same styles as it currently has but add display block so it doesn't run into it. Or do you mean skimming with a screen reader?

Reply:
yeah, I meant with a screen reader. It seems odd to me that we would go for a visual distinction between title and subtitle, but try to remove that semantic distinction for screen-readers? Does a `p` in the `header` really get so lost that we want to provide really long `h1`s?

Reply:
nope. I think you are right. The span would mostly only provide more important keywords for Google, and would provide very little for the most part for screen readers. I didn't realize the h2 was frowned upon in a <header> originally.